src/components/verify-phone/VerifyPhoneForm/index.jsx

```diff
@@ -113,7 +113,7 @@ function VerifyPhoneForm() {
           </CountdownCircleTimer>
 
           <ResendButton onClick={() => setResendCode(!resendCode)}>
-            Did not receive code? send again
+            Didn&apos;t receive a code? Send it again
```

Comment:
&apos isn not a valid html character if we want to use an apostrophe use it like this
```suggestion
           { "Didn&apos;t receive a code? Send it again"}
```

please check if there anywhere else that uses &apos and fix it as suggested.

Reply:
sorry that didn't work!

Reply:
try this  {`
            Didn't receive a code? Send it again`}